tests/unit/XMLDataFormatterTest.php

```diff
@@ -74,4 +76,51 @@ public function testConvertStringToArrayMultipleEntitiesException()
         $formatter = new XMLDataFormatter();
         $formatter->convertStringToArray($inputXML);
     }
+
+    /**
+     * Data provider for trailing slash configuration
+     */
+    public function trailingSlashProvider(): array
+    {
+        return [
+            'without trailing slash' => [false],
+            'with trailing slash' => [true],
+        ];
+    }
+
+    /**
+     * Tests wrapper output of {@link XMLDataFormatter::convertDataObjectWithoutHeader()}
+     *
+     * @param bool $addTrailingSlash - Whether to add a trailing slash to the API endpoint
+     * @dataProvider trailingSlashProvider
+     */
+    public function testConvertDataObjectWithoutHeaderClassNameAttribute(bool $addTrailingSlash): void
```

Comment:
```suggestion
    /**
     * Tests wrapper output of {@link XMLDataFormatter::convertDataObjectWithoutHeader()}
     *
     * @dataProvider provideConvertDataObjectWithoutHeaderClassNameAttribute
     */
    public function testConvertDataObjectWithoutHeaderClassNameAttribute(bool $addTrailingSlash): void
```
Don't need docblock type as params strongly typed

Reply:
ok, if that is the convention - I find it useful to add some extra info on parameters